chipsec/cfg/pch_3xx.xml

```diff
@@ -71,13 +71,20 @@ XML configuration file for the 300 series PCH
     </register>
 
     <!-- Power Management Controller -->
-    <register name="ABASE" type="pcicfg" bus="0" dev="0x1f" fun="2" offset="0x40" size="4" desc="ACPI Base Address">
-      <field name="STYPE" bit="0" size="1" desc="Space Type (always 1 - I/O space)"/>
-      <field name="BA"    bit="8" size="8" desc="Base Address"/>
+     <register name="ABASE" type="mm_msgbus" port="0x88" offset="0x27B4" size="4" desc="ACPI Base Address">
+      <field name="MASK" bit="18" size="6" desc="MASK for lower 6 bits"/>
+      <field name="BA"    bit="2" size="14" desc="Base Address"/>

```

Comment:
This is missing the ACPIBADE field (bit 0, size 1), though I don't think it'll be used for now.